### PR TITLE
Compile lib to dist folder

### PR DESCRIPTION
**Pull request by ahennr**
This PR enhances build procedure of `geostyler` to enable its usage in other projects (e.g. `react-geo`). 

In particular:

* compliled output (using `tsc`) and CSS files are exposed in `dist` folder by running `npm run build`
* Named exports of each included module (as in `react-geo`)

### CODE REVIEW

File: src/Component/Filter/AttributeCombo/AttributeCombo.tsx [hunk 1]
```diff
@@ -71,7 +70,7 @@ class AttributeCombo extends React.Component<AttributeComboProps, AttributeCombo
     return (
       <div className="gs-attr-combo">
 
-        <FormItem label={this.props.label} colon={false} >
+        <Form.Item label={this.props.label} colon={false} >
```
Comment (KaiVolland) [inline src/Component/Filter/AttributeCombo/AttributeCombo.tsx]: Is this change required for this PR to work. If not please split this into another PR.

File: tsconfig.json [hunk 2]
```diff
@@ -20,11 +20,12 @@
   },
   "exclude": [
     "node_modules",
-    "build",
```
Comment (KaiVolland) [inline tsconfig.json]: Please leave the "build" as it will be used for other stuff like styleguidist.

**Comment by KaiVolland** [comment]
Thanks please merge.

**Comment by ahennr** [comment]
:top:
